### DISCUSSION TITLE
🐞 Bug Report: Closed Tabs "Widget

### DISCUSSION THREAD

**Discussion by KaeligBrl** [elementor/elementor#21223] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hello, 

I can't close a tab when clicking on it again.

I have Elementor Pro

### Describe the solution you'd like

I would like to be able to close the active tab as soon as I click on it again.

**Comment by BaregoS** [comment]
Would love to have an answer for this as well.
